components/network/isc-dhcp/files/isc-dhcp-relay.xml

```diff
@@ -69,7 +69,7 @@
 	<exec_method type='method' name='stop' exec=':kill' 
 		timeout_seconds='60'/>
 
-	<exec_method type='method' name='refresh' exec=':kill -HUP' 
+	<exec_method type='method' name='refresh' exec=':true' 
```

Comment:
Why do we remove refresh method?

Reply:
Although upstream says that `dhcpd` is [unable to reload itself](https://kb.isc.org/docs/aa-00335), I guess what they mean is that `dhcpd` can't catch the signal and handle it on it's own. But Oracle [got bitten by it](https://github.com/oracle/solaris-userland/commit/22d8d271599a6500d912ac98c1060aa771be874a), it seems.

Trying it manually seems to work fine:
```
{global} newman@lenovo:~ $ svcs -vp svc:/network/dhcp/server:ipv4
STATE          NSTATE        STIME    CTID   FMRI
online         -             14:31:31    351 svc:/network/dhcp/server:ipv4
               14:31:31     4807 dhcpd
{global} newman@lenovo:~ $ pfexec truss -p $(pgrep dhcpd)
pollsys(0xFFFFFD7FFFDFF7E0, 2, 0x00000000, 0x00000000) (sleeping...)
    Received signal #1, SIGHUP, in pollsys() [default]
      siginfo: SIGHUP pid=4812 uid=0
pollsys(0xFFFFFD7FFFDFF7E0, 2, 0x00000000, 0x00000000) Err#4 EINTR
{global} newman@lenovo:~ $ svcs -vp svc:/network/dhcp/server:ipv4
STATE          NSTATE        STIME    CTID   FMRI
online         -             14:31:54    352 svc:/network/dhcp/server:ipv4
               14:31:54     4821 dhcpd
```
SMF handled PID transition correctly.

Maybe if refresh timeout is too short to wait for a bigger setup to load, then it won't do the handover?

Reply:
OK, let's fetch this change.